pyproject.toml

```diff
@@ -0,0 +1,150 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
```

Comment:
Is there a reason for using this particular flit_core version? v3.8 would allow recursive pattern matching in includes and excludes.

Reply:
I think that came from your original work from last year, perhaps? Going too recent could cause issues for third-party packagers - but it's 5 months old now, so I'd say it's probably okay? Include/exclude lists would, as you say, be much nicer. Happy to update if you want me to.

Reply:
Please do.